package.json

```diff
@@ -53,16 +53,34 @@
 	"contributes": {
 		"languages": [
 			{
-				"id": "javascript",
```

Comment:
I think we still need `javascript`, `typescript` langs to support higligting cases like:
`AppContainer.ts`
```
import { hbs } from '@glimmerx/component';

const AppContainer: any = hbs`
<section class="text-gray-600 body-font">
    <div class="container px-5 py-24 mx-auto">
        <div class="xl:w-1/2 lg:w-3/4 w-full mx-auto text-center">
            {{yield}}
        </div>
    </div>
</section>
`;


export default AppContainer;
```

Reply:
Checking it...

Reply:
Looks like it's working :)